script/index.ts

```diff
@@ -15,6 +15,7 @@ import {
 import { config, getNetwork, getSigningClient } from './shared/config';
 import { loadContext } from './shared/context';
 import { CONTAINER, Dependencies } from './shared/ioc';
+import { registerCmd } from './commands/register';
```

Comment:
import { updateCmd } from './commands/update';

Reply:
I guess these didn't get added right after i renamed it